### PR TITLE
Add api.rhods-idh.dev.datahub.redhat.com server configuration

### PR DESCRIPTION
**Pull request by rimolive**
## This Pull Request implements

Server definition api.rhods-idh.dev.datahub.redhat.com added in clusters configuration.

### CODE REVIEW

File: manifests/overlays/prod/secrets/clusters/api.rhods-idh.dev.datahub.redhat.com.enc.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    namespaces: dh-stage-presto
+    server: https://api.rhods-idh.dev.datahub.redhat.com:6443
+    shard: "1"
```
Comment (anishasthana) [inline manifests/overlays/prod/secrets/clusters/api.rhods-idh.dev.datahub.redhat.com.enc.yaml:15]: Can you put this on shard "0" instead?

**Review by anishasthana — COMMENTED** [review]
Did you create the argocd namespace and SA on this cluster?

**Comment by rimolive** [comment]
@anishasthana Yes, argocd namespace and SA are set up in the cluster

File: manifests/overlays/prod/secrets/clusters/api.rhods-idh.dev.datahub.redhat.com.enc.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+kind: Secret
+metadata:
+    name: api.rhods-idh.dev.datahub.redhat.com
+    labels:
+        argocd.argoproj.io/secret-type: cluster
+    annotations:
+        managed-by: argocd.argoproj.io
+type: Opaque
+stringData:
+    name: api.ocp4.prod.psi.redhat.com
```
Comment (anishasthana) [inline manifests/overlays/prod/secrets/clusters/api.rhods-idh.dev.datahub.redhat.com.enc.yaml:11]: This name is wrong

**Review by anishasthana — CHANGES_REQUESTED** [review]

**Review by anishasthana — COMMENTED** [review]
/lgtm

**Review by anishasthana — APPROVED** [review]
/lgtm

**Pull request merged**